vr-wordle/src/Components/Column.js

```diff
@@ -1,10 +1,37 @@
-import React from "react";
-import { useGLTF } from "@react-three/drei";
-import ColumnContainer from "./ColumnContainer.js";
+import React, { useRef } from 'react';
+import { useLoader } from '@react-three/fiber';
+import { TextureLoader } from 'three';
+import { useGLTF } from '@react-three/drei';
+import ColumnContainer from './ColumnContainer.js';
 
 export default function Model({ position, guessIndex, setGuess }) {
+  const [colorMap, normalMap, roughnessMap, aoMap] = useLoader(TextureLoader, [
+    '/textures/aerial_rocks_02_diff_2k.jpg',
+    '/textures/aerial_rocks_02_nor_dx_2k.jpg',
+    '/textures/aerial_rocks_02_rough_2k.jpg',
+    '/textures/aerial_rocks_02_ao_2k.jpg',
+  ]);
```

Comment:
we need to use the current texture

>
```suggestion
  const [colorMap] = useLoader(TextureLoader, [
    '/textures/mossy-rock.png',
  ]);
```